### ISSUE TITLE
executor: implement executor

### ISSUE THREAD

**Issue by TimSatke** [tomarrell/lbadd#42]
An executor is the connection between the database input service and the database layer. Somewhere, an SQL statement is parsed and converted to the IR, which are commands that the executor can execute.
The executor can then execute the appropriate methods and functions on the database in order to achieve, what the command describes.

**Comment by TimSatke** [comment]
Depends on #43

**Comment by TimSatke** [comment]
Depends on #128

**Comment by TimSatke** [comment]
Closing in favor of #177
